wizard-frontend/src/App.tsx

```diff
@@ -0,0 +1,56 @@
+import { Toaster } from "@/components/ui/toaster";
+import { Toaster as Sonner } from "@/components/ui/sonner";
+import { TooltipProvider } from "@/components/ui/tooltip";
+import {
+  QueryClient,
+  QueryClientProvider,
+  useQuery,
+} from "@tanstack/react-query";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Homepage from "./pages/Homepage";
+import NotFound from "./pages/NotFound";
+import { UserDataContext } from "./components/UserDataContext";
+
+const queryClient = new QueryClient();
+
+const InnerApp = () => {
+  const { error, isLoading, data } = useQuery({
+    queryKey: ["userIsReturning"],
+    queryFn: () =>
+      fetch(window.location.href + "api/v1/is-returning").then(async (res) =>
```

Comment:
AI said to use https://developer.mozilla.org/en-US/docs/Web/API/Location/origin instead of `href`, cause `href` may or may **not** contain a trailling `/`.

Reply:
Can we also get a global/const json object with all the backend routes? Something like:

```js
const ALL_THE_ROUTES = {
    isReturning: "/api/v1/is-returning,
};
```

Something along these lines, so we can have a central point to define all of this strings (muh grpc).